packages/connector-types/src/ComponentConnector.ts

```diff
@@ -0,0 +1,29 @@
+import { QueryPage } from './Connector.Shared';
+
+/**
+ * @experimental This is still experimental and might change in future releases.
+ */
+export type ComponentPage = QueryPage<Component>;
+
+/**
+ * @experimental This is still experimental and might change in future releases.
+ */
+export interface Component {
+    id: string;
+    name: string;
+    defaultComponentDimensions: {
```

Comment:
```suggestion
    defaultDimensions: {
```
Isn't the name redundant?

Reply:
Yes, but this aligns closely with the backend so we keep it in line